superset-frontend/plugins/plugin-chart-table/src/controlPanel.tsx

```diff
@@ -290,8 +290,7 @@ const config: ControlPanelConfig = {
             },
           },
         ],
-        isFeatureEnabled(FeatureFlag.DASHBOARD_CROSS_FILTERS) ||
-        isFeatureEnabled(FeatureFlag.DASHBOARD_NATIVE_FILTERS)
+        isFeatureEnabled(FeatureFlag.DASHBOARD_CROSS_FILTERS)
```

Comment:
This condition ought to be disregarded as it is constantly satisfied due to its association with an 'OR' condition before.

Reply:
Makes sense. Done.